signing_service/signing_service.go

```diff
@@ -7,7 +7,6 @@ import (
 
```

Comment:
are we still using the "signing_service" for anything, or can we remove this code...?

Reply:
I believe we can safely delete it now.